LEAF_Request_Portal/sources/Form.php

```diff
@@ -1126,20 +1126,37 @@ private function writeDataField($recordID, $key, $series)
             return 0;
         }
 
+        $validMetadataFields = array(
+            "firstName" => 1,
+            "lastName" => 1,
+            "middleName" => 1,
+            "email" => 1,
+            "userName" => 1,
+        );
+        $metadata = array();
+        if(isset($_POST[$key . "_metadata"])) {
+            $postMetadata = json_decode($_POST[$key . "_metadata"], true);
```

Comment:
For backward compatibility, we cannot rely on the user providing this via HTTP POST.

See line ~1099 where logic should be included to handle orgchart_employee formats. Since this format typically only writes the empUID, we'll need to create the metadata based on a server-side empUID lookup.

From the API perspective, interacting with the metadata must be completely transparent to ensure this doesn't conflict with customizations.